### ISSUE TITLE
Schema language descriptions

### ISSUE THREAD

**Issue by joemcbride** [graphql-dotnet/parser#4]
I'm not sure that the parser handles these yet.

https://github.com/graphql/graphql-js/pull/464

```
# Marks an element of a GraphQL schema as no longer supported.
directive @deprecated(
  # Explains why this element was deprecated, usually also including a suggestion
  # for how to access supported similar data. Formatted in
  # [Markdown](https://daringfireball.net/projects/markdown/).
  reason: String = "No longer supported"
) on FIELD_DEFINITION | ENUM_VALUE
```

**Comment by sungam3r** [comment]
I encountered the same problem with the descriptions. What are your plans for this issue and PR?

**Comment by sungam3r** [comment]
Issue is outdated.

**Comment by joemcbride** [comment]
Fixed in #26
